player.cpp

```diff
@@ -84,9 +84,12 @@ int main(int argc, char* argv[])
 {
 	int argbase;
 	UINT8 retVal;
-	FileLoader fLoad;
+	FileLoader _fLoad;
+	FileLoader *fLoad = &_fLoad;
```

Comment:
Please use don't declare `FileLoader` and `FileLoader*` just to always use the latter.  
Use just `FileLoader fLoad` and use `&` operators where necessary.